### PR TITLE
return500OnFailure optional parameter

### PR DESCRIPTION
**Pull request by jankowiakmaria**

### CODE REVIEW

**Comment by andyroyle** [comment]
Maybe make this more general so that you can specify the failure statusCode as config:

e.g.

```
options: {
  failureStatusCode: 500,
  monitors: [...],
  default: 0
}
```

Default should probably still be 200.

**Comment by jankowiakmaria** [comment]
I've made it more general and I've added description in readme file.

File: index.js [hunk 1]
```diff
@@ -15,6 +15,13 @@ var validateOptions = function(options){
   return result.value;
 };
 
+var statusCode = function(status){
+  if(status == 'Failed'){
```
Comment (andyroyle) [inline index.js]: should really be '==='

**Comment by jankowiakmaria** [comment]
@andyroyle anything else to fix (in this PR)? :)

**Comment by andyroyle** [comment]
published in 2.3.0

**Comment by andyroyle** [comment]
I've added you as a collaborator too :+1:

**Comment by jankowiakmaria** [comment]
Cool, thanks.

**Pull request merged**